### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by OleksandrSPV**
Updated javascript packages
Moved node version to lts

### CODE REVIEW

**Comment by OleksandrSPV** [comment]
@OlhaPeculiar Could you check it please here https://fortify-tools.vercel.app/
Thanks.

**Comment by OlhaPeculiar** [comment]
At the moment everything is working correctly on https://fortify-tools.vercel.app/

**Comment by OleksandrSPV** [comment]
@OlhaPeculiar Could you check please here https://fortify-tools.vercel.app/

**Comment by OlhaPeculiar** [comment]
At the moment everything is working correctly on https://fortify-tools.vercel.app/

File: yarn.lock [hunk 1]
```diff
@@ -5434,60 +4778,62 @@ util@^0.12.4, util@^0.12.5:
...
+  resolved "https://registry.yarnpkg.com/vite/-/vite-5.4.11.tgz#3b415cd4aed781a356c1de5a9ebafb837715f6e5"
+  integrity sha512-c7jFQRklXua0mTzneGW9QVyxFjUgwcihC4bXEtujIo2ouWCe1Ajt/amn2PCxYnhYfd5k09JX3SB7OYWFKYqj8Q==
   dependencies:
     esbuild "^0.21.3"
     postcss "^8.4.43"
     rollup "^4.20.0"
   optionalDependencies:
     fsevents "~2.3.3"
 
+vite@^6.0.3:
```
Comment (donskov) [inline yarn.lock:4826]: Maybe we will find an option not to use 2 versions of this module?

Comment (OleksandrSPV) [reply yarn.lock]: @donskov check please https://github.com/PeculiarVentures/fortify-tools/pull/231/commits/45cdf93d524a57d53a599bef5a854fbd7382b7cd

**Pull request merged**